cabal.project

```diff
@@ -10,38 +10,17 @@ repository cardano-haskell-packages
     c00aae8461a256275598500ea0e187588c35a5d5d7454fb57eac18d9edb86a56
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
--- While using SRPs one can obtain the `--sha256` value for a package
--- by first setting it to some random value and letting the tooling
--- tell you what it should be, for example, using `nix develop` will
--- throw an error with the correct value to use or even better you
--- can use `nix-prefetch-git`:
---
--- $ nix-shell -p nix-prefetch-git
--- $ nix-prefetch-git https://github.com/intersectmbo/formal-ledger-specifications --rev <GIT_SHA> | jq .hash
 source-repository-package
   type: git
   location: https://github.com/IntersectMBO/formal-ledger-specifications.git
+  subdir: generated
   -- !WARNING!:
   -- MAKE SURE THIS POINTS TO A COMMIT IN `MAlonzo-code` BEFORE MERGE!
-  subdir: generated
-  --sha256: sha256-vrxKI3I1kwt+XiHk+UAWm49veRPTWkVPOvwNzVxuFs8=
```

Comment:
Just FIY Carlos, we have[ a CI check ](https://github.com/IntersectMBO/cardano-ledger/blob/master/.github/workflows/haskell.yml#L493) that looks at  this part of the cabal file to validate that the formal-ledger-specifications tag is correct - and the check is quite rigid, so it's good to pay attention when modifying this `source-repository-package`   ( I should have probably added a warning comment last time it broke).  
I checked now that it's fine with these changes, just wanted to make you aware of the existence of this check.